Upstream: yes
Reason: Project tests that it responds properly to a revoked
certificate, but the certificate they test is expired, causing a
different exception. Ticket was created upstream.
Url: https://github.com/wbond/oscrypto/issues/45

diff --git a/tests/test_tls.py b/tests/test_tls.py
index c171724..e4581b4 100644
--- a/tests/test_tls.py
+++ b/tests/test_tls.py
@@ -111,6 +111,7 @@ class TLSTests(unittest.TestCase):
         html = connection.read_until(re.compile(b'</html>', re.I))
         self.assertNotEqual(None, re.search(b'</html>', html, re.I))
 
+    @unittest.skip('Revoked certificate is expired')
     @connection_timeout()
     def test_tls_connect_revoked(self):
         if _backend == 'mac':
